doc/users/next_whats_new/2019-03-25-mplot3d-projection.rst

```diff
@@ -0,0 +1,5 @@
+``Axes3D`` now preserves right angles when rotating
+---------------------------------------------------
+
+:class:`~mpl_toolkits.mplot3d.Axes3D` no longer stretches the plot in the x
+axis after projecting.
```

Comment:
This is a bit misleading. It isn't always a stretch in the x axis. Look at some of the image diffs, particularly the bar3d rotated one, and sometimes the stretching was in other directions. This is also misleading to some users because they may not be explicitly rotating the figure, but this still impacts them. This impacts just about anybody who has used mplot3d.

Reply:
It's a stretch in the window x axis (ie, the axis used after projection). You're right though, the wording is bad.